pkg/operator/encryption/controllers.go

```diff
@@ -34,6 +37,7 @@ func NewControllers(
 	apiServerInformer configv1informers.APIServerInformer,
 	kubeInformersForNamespaces operatorv1helpers.KubeInformersForNamespaces,
 	secretsClient corev1.SecretsGetter,
+	configmapsClient corev1.ConfigMapsGetter,
```

Comment:
We need to add a configmap client when creating the resource syncer even though we don't need to sync configmaps in the encryption controller. Any suggestion on how this could be improved?